File: strings/557. Reverse Words in a String III/reverseWords.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    s = s.split(' ');
    
    for (let i = 0; i < s.length; i++) {
        let reversed = "";
        for (let j = s[i].length - 1; j >= 0; j--) {
            reversed += s[i][j];
        }
        s[i] = reversed;
    }

    return s.join(' ');
};

File: strings/557. Reverse Words in a String III/reverseWords.py
class Solution(object):
    def reverseWords(self, s):
        """
        :type s: str
        :rtype: str
        """
        s = s.split(' ')
        
        # for word in s
        for i in range(len(s)):
            temp = ""
            # for char in word in reverse
            for j in range(len(s[i]) - 1, -1, -1):
                temp += s[i][j]
            s[i] = temp
        
        return ' '.join(s)